src/strawberry_sqlalchemy_mapper/field.py

```diff
@@ -333,8 +341,14 @@ def _get_query(s: Session):
                     else:
                         query = s.query(model)
 
-                    if field.keyset is not None:
-                        query = query.order_by(*field.keyset)
+                    order_by = []
+                    if order := kwargs.get("order"):
+                        order_by.extend([getattr(model, o) for o in order])
+                    if field.keyset:
+                        order_by.extend(field.keyset)
+
+                    if order_by:
+                        query = query.order_by(*order_by)
```

Comment:
Ah! No need to add this "order" column yet since we don't have a fully fleshed API yet. I think you can just do:

```python
if after := kwargs.get("after"):
  order_by.extend(sqlakeyset.unserialize_bookmark(after).place)
```

And then specify where field.keyset is defined that it's just the default ordering, and later pages grab the order from the cursor.